object/relations/relations.go

```diff
@@ -44,42 +27,56 @@ var (
 	ErrNoSplitInfo = errors.New("no split info")
 )
 
-// ListAllRelations return all related phy objects for provided root object ID in split-chain order.
-// Result doesn't include root object ID itself. If linking object is found its id will be the last one.
-func ListAllRelations(ctx context.Context, rels Relations, cnrID cid.ID, rootObjID oid.ID, tokens Tokens) ([]oid.ID, error) {
-	return ListRelations(ctx, rels, cnrID, rootObjID, tokens, true)
+// HeadExecutor describes methods to get object head.
+type HeadExecutor interface {
+	ObjectHead(ctx context.Context, containerID cid.ID, objectID oid.ID, prm client.PrmObjectHead) (*client.ResObjectHead, error)
+}
+
+// SearchExecutor describes methods to search objects.
+type SearchExecutor interface {
+	ObjectSearchInit(ctx context.Context, containerID cid.ID, prm client.PrmObjectSearch) (*client.ObjectListReader, error)
+}
+
+// Executor describes all methods required to find all siblings for object.
+type Executor interface {
+	HeadExecutor
+	SearchExecutor
 }
 
-// ListRelations return all related phy objects for provided root object ID in split-chain order.
+// Get returns all related phy objects for provided root object ID in split-chain order.
+// If linking object is found its id will be the last one.
```

Comment:
will it? i would also describe returned values more details

Reply:
You are right, documentation updated